tools/shell/embedded_shell.cpp

```diff
@@ -323,6 +333,34 @@ void EmbeddedShell::setNumThreads(const std::string& numThreadsString) {
     } catch (Exception& e) { printf("%s", e.what()); }
 }
 
+void EmbeddedShell::setMaxRows(const std::string& maxRowsString) {
+    uint64_t parsedMaxRows = 0;
+    try {
+        parsedMaxRows = stoull(maxRowsString);
+    } catch (std::exception& e) {
+        printf("Cannot parse '%s' as number of rows. Expect integer.\n", maxRowsString.c_str());
+        return;
+    }
+    if (maxRows == 0) {
+        maxRows = defaultMaxRows;
+    } else {
+        maxRows = parsedMaxRows;
+    }
```

Comment:
You can inline this if else statement.
maxRows = maxRows == 0 ? defaultMaxRows : parsedMaxRows;
What if the parsedMaxRows is also 0?